{"uuid": "e508ac0b-8dd3-43ed-b8a1-a767331bf884", "children": ["b31e560c-1d87-4e53-b73f-dbccc762faed"], "befores": [{"name": "driver_class", "status": "broken", "statusDetails": {"message": "_pytest.config.exceptions.UsageError: --driver must be specified\n", "trace": "  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\pytest_selenium\\pytest_selenium.py\", line 181, in driver_class\n    raise pytest.UsageError(\"--driver must be specified\")\n"}, "start": 1678354336030, "stop": 1678354336030}], "start": 1678354336030, "stop": 1678354410445}